libretro/libretro_core_options.h

```diff
@@ -224,7 +237,11 @@ struct retro_core_option_definition option_defs_us[] = {
             {"Sync", NULL},
             {NULL, NULL},
         },
+#ifndef HAVE_OPENGLES
         "Async"
```

Comment:
I think this logic is slightly wrong - see https://github.com/libretro/mupen64plus-libretro-nx/pull/160/files

Reply:
This is just copied from here: https://github.com/libretro/mupen64plus-libretro-nx/blob/develop/libretro/libretro.c#L274

I would suggest that this MR simply replicates the current state and the other merge request deals with the behaviour change, but I'm open to doing it here if that's what's desired.